anyway/views/user_system/api.py

```diff
@@ -574,3 +574,21 @@ def update_user_org(user_email: str, org_name: str) -> Response:
 
     db.session.commit()
     return Response(status=HTTPStatus.OK)
+
+
+@roles_accepted(BE_CONST.Roles2Names.Admins.value)
+def delete_user(email: str) -> Response:
+    user = get_user_by_email(db, email)
+    if user is None:
+        return return_json_error(Es.BR_USER_NOT_FOUND, email)
+
+    # Delete user roles
+    user.roles = []
+
+    # Delete user organizations membership
+    user.organizations = []
```

Comment:
Hi @BarVolunteering , does this line removes the user entry from `users_to_organizations` table?

Reply:
Yes